assets/html/scss/documenter/_ansicolors.scss

```diff
@@ -0,0 +1,181 @@
+
+$ansi-black:  #111 !default;
+$ansi-red:    #944 !default;
+$ansi-green:  #073 !default;
+$ansi-yellow: #870 !default;
+$ansi-blue:   #15a !default;
+$ansi-magenta:#94a !default;
+$ansi-cyan:   #08a !default;
+$ansi-white:  #ddd !default;
+
+$ansi-light-black:  #888 !default;
+$ansi-light-red:    #d57 !default;
+$ansi-light-green:  #2a5 !default;
+$ansi-light-yellow: #d94 !default;
+$ansi-light-blue:   #08d !default;
+$ansi-light-magenta:#b8d !default;
+$ansi-light-cyan:   #0bc !default;
+$ansi-light-white:  #eee !default;
+
+span {
```

Comment:
Adding `documenter-ansi` is fine, but what is its primary purpose? In general, the looser the style specifications, the easier they are to customize (i.e. override).

What I'm concerned about is the condition on sticking `documenter-ansi`. By the new default, `ansicolor` is `true` in most cases. Do you want to make sure that the escape codes are actually included?

Reply:
It's just that the names `sgr1` etc. are very short and generic, so it feels like it might be a good idea to namespace them, to make sure it doesn't conflict with something in the future (but it's good to keep them short).

Actually, something like `.ansi` or `.ansicolor` would be a better class name.

Reply:
`.ansi` is added to the `<code>`'s class now. However, it is also attached in the case of `ansicolor=false` for now.:sweat_smile: